code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -567,85 +501,67 @@
 	model_type = list(/obj/item/robot_model/medical, /obj/item/robot_model/syndicate_medical)
 	model_flags = BORG_MODEL_MEDICAL
 
-/obj/item/borg/upgrade/processor/action(mob/living/silicon/robot/R, user = usr)
-	. = ..()
-	if(.)
-		var/obj/item/borg/upgrade/processor/U = locate() in R
-		if(U)
-			to_chat(user, span_warning("This unit is already equipped with a surgical processor module!")) //check to see if we already have this module
-			return FALSE
-		var/obj/item/surgical_processor/SP = new(R.model)
-		R.model.basic_modules += SP
-		R.model.add_module(SP, FALSE, TRUE)
-
-/obj/item/borg/upgrade/processor/deactivate(mob/living/silicon/robot/R, user = usr)
-	. = ..()
-	if (.)
-		var/obj/item/surgical_processor/SP = locate() in R.model
-		R.model.remove_module(SP, TRUE)
+	items_to_add = list(/obj/item/surgical_processor)
 
 /obj/item/borg/upgrade/ai
-	name = "B.O.R.I.S. module"
+	name = "B.O.borg.I.S. module"
```

Comment:
search and replace fail.

Reply:
Lmao how I dont recall searching for just R but robot/R, oh well